heaps/min_heap.py

```diff
@@ -10,69 +10,94 @@ def __str__(self):
     def __repr__(self):
         return str(self.value)
 
-
-
 class MinHeap:
 
     def __init__(self):
         self.store = []
 
-
-    def add(self, key, value = None):
+    def add(self, key, value=None):
         """ This method adds a HeapNode instance to the heap
             If value == None the new node's value should be set to key
-            Time Complexity: ?
-            Space Complexity: ?
+            Time Complexity: O(logn)
+            Space Complexity: O(1)
         """
```

Comment:
👍 However because heap_up is recursive the call stack adds O(log n) space complexity.